packages/website/ts/containers/portal_onboarding_flow.ts

```diff
@@ -11,22 +11,39 @@ interface PortalOnboardingFlowProps {}
 interface ConnectedState {
     stepIndex: number;
     isRunning: boolean;
+    userAddress: string;
+    providerType: ProviderType;
+    injectedProviderName: string;
+    blockchainIsLoaded: boolean;
+    hasBeenSeen: boolean;
 }
 
 interface ConnectedDispatch {
-    onClose: () => void;
+    setIsRunning: (isRunning: boolean) => void;
```

Comment:
should we prefix these with `update` so that we are consistent with the associated redux types?